src/adapters/store.js

```diff
@@ -18,9 +18,9 @@ function Store() {
   })
 }
 
-Store.prototype.getAll = async function() {
+Store.prototype.getAllPois = async function() {
   return new Promise((resolve, reject) => {
-    abstractStore.getAll().then((masqData) => {
+    abstractStore.getAllPois().then((masqData) => {
```

Comment:
i have moved from then to async await syntax. can you update this part with the new syntax

Reply:
I prefer to refactor the code in another context rather than do it here